pages/common/q.md

```diff
@@ -0,0 +1,27 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query .csv file by specifying the delimeter as ',':
+
+`q -d',' "SELECT * from {{path/to/file}}"`
+
+- Query .tsv file:
+
+`q -t "SELECT * from {{path/to/file}}"`
+
+- Query file with header row:
+
+`q -d{{delimiter}} -H "SELECT * from {{path/to/file}}"`
+
+- Read data from stdin; '-' in the query represents the data from stdin:
+
+`{{output}} | q "select * from -"`
+
+- Join two files:
+
+`q "SELECT * FROM {{path/to/file}} f1 JOIN {{path/to/other_file}} f2 ON (f1.c1 = f2.c1)"`
+
+- Format output using an output delimeter with an output header line:
+
+`q -D{{delimiter}} -O "SELECT * from {{path/to/file}}"`
```

Comment:
`-O` is an optionless flag.
It either prints out the header line from the file you are reading,
or the column names via aliasing in the query.